README.md

```diff
@@ -68,26 +68,34 @@ cd orbit.<your_extension_name>
 mv orbit/ext_template orbit/<your_extension_name>
 ```
 
-- Set up a symbolic link from Orbit to this directory.
-This makes it convenient to index the python modules and look for extensions shipped with Isaac Sim and Orbit.
+- Define the following environment variable to specify the path to your Orbit installation:
 
 ```bash
-ln -s <your_orbit_path> _orbit
```

Comment:
If you don't have this then setting up VSCode doesn't work because it looks for this path for configuring settings.json

Reply:
I'll try to find a solution around this. The problem here is that there would be a symlink loop when users add this template to `source/extensions`.

Reply:
Fixed with the new commit. We pass an argument to `setup_python_env` to bypass this problem.